src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private User user;
+
+    @BeforeAll
+    public static void init() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void sample() {
+        user = new User();
+        user.setLogin("login");
+        user.setAge(54);
+        user.setPassword("password");
```

Comment:
```suggestion
        user = new User();
        user.setLogin("login");
        user.setAge(54);
        user.setPassword("password");
```

you have constructor in User class, so you can simplify creating new user everywhere

and if you use it everywhere, you can initialize it once in @BeforeAall

Reply:
If i use @BeforeAll, it works wrong